Doc/library/venv.rst

```diff
@@ -371,7 +371,7 @@ creation according to their needs, the :class:`EnvBuilder` class.
     .. method:: create_git_ignore_file(context)
 
        Creates a ``.gitignore`` file within the virtual environment that causes
-       the entire directory to be ignored by the ``git`` source control manager.
+       the entire directory to be ignored by the Git source control manager.
```

Comment:
IMO, most folks seem to call it a VCS instead of SCM these days, but I digress

Reply:
In the docs, we have:

* 2 "version control system" + 3 VCS
* 1 "source control manager" + 5 SCM

Too close to be decisive.

I don't mind, feel free add a a suggest if you want to change and I'll accept it :)